README.md

```diff
@@ -39,7 +39,7 @@ If your AWS credentials are in the default profile, you can omit the `AWS_PROFIL
 
 With all the credentials in place, running
 ```
-$ s3same --repo some-repo-name --owner some-user --pro
+$ s3same some-repo-name --owner some-user --pro
```

Comment:
The usage line above (`Usage: s3same [OPTIONS] REPO`) implies that the repo name should come after the options, not before. Does it matter?